Chapter 1/1 - Creating a Window/Window.cs

```diff
@@ -6,7 +6,7 @@ namespace LearnOpenGL_TK
 {
     // This is where all OpenGL code will be written.
     // OpenTK allows for several functions to be overriden to extend functionality; this is how we'll be writing code.
-    internal class Window : GameWindow
+    public class Window : GameWindow
```

Comment:
Although making all classes internal isn't that bad I didn't mean to do it in d2da622674aef1cbe6671316549dd3ece771c19e.
The internal modifier was added by rider; I'm now removing it.